iotedgedev/modules.py

```diff
@@ -163,3 +168,31 @@ def deploy_device_configuration(
             self.output.error(
                 "There was an error deploying the configuration. Please make sure your IOTHUB_CONNECTION_STRING and DEVICE_CONNECTION_STRING Environment Variables are correct.")
 
+class ModulesProcessorFactory(Modules):
+    
+    def factory(self, module_language):
```

Comment:
If you are going to use Factory in the class name, then rename the method to "get"